src/parser/jobs/ast/ArcanaUndrawUsage.js

```diff
@@ -0,0 +1,97 @@
+import React, {Fragment} from 'react'
+
+import ACTIONS from 'data/ACTIONS'
+// import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import {ActionLink} from 'components/ui/DbLink'
+
+const OGCD_ARCANA_REMOVAL = [
+	ACTIONS.UNDRAW_SPREAD.id,
+	ACTIONS.EMPTY_ROAD.id,
+	ACTIONS.UNDRAW.id,
+]
+
+export default class ArcanaUndrawUsage extends Module {
```

Comment:
This module seems a bit suss to me.

I'm no healer main, but while I'd certainly _recommend_ using a shadowflare macro on SMN, it's not something I'd necessarily raise an issue about.

Are we 100% sure that not using a macro is always a bad idea?

Reply:
So the funny thing is that if you use the given actions, you can't remove cards in your spread while casting another spell. There's no animation lock but it still delays the next GCD by a fraction of a second, even if you mash it. I would value getting more opinions on this, but that's my observation.